src/App.vue

```diff
@@ -100,21 +99,6 @@ export default defineComponent({
     emitter.on('presentLoader', this.presentLoader);
     emitter.on('dismissLoader', this.dismissLoader);
   },
-  async mounted() {
-    if(this.productIdentifications.length <= 0) {
```

Comment:
We need to fetch the product identification options from dxp on app mount to get the latest values for primary and secondary identifier.